add more abbreviations to glossary (#3)
OlympusDAO-Education/Documentation: glossary.md

higher BCV means a lower discount for bonders and higher inflation by the
protocol. A lower BCV means a higher discount for bonders and lower inflation by
the protocol.

### PCV

> Protocol Controlled Value, is the amount of funds the treasure owns and
> controlls. The more PCV the better for the protocol and its users.

Reviewer: ```suggestion
Protocol Controlled Value, is the amount of funds the treasury owns and
controls. The more PCV the better for the protocol and its users.
```

### RFV

Risk Free Value, is the amount of funds the treasury guarantees to use for
backing $OHM. For instance $DAI bonds are most efficient for the treasury since

> 100% of the $DAI provided can be accounted for as risk free value.

Reviewer: I think adding the $DAI bond example here does not help clarifying the term. But I couldn't come out with a concise explanation either

Reviewer: ```suggestion
backing $OHM.
```

Reviewer: I think there are different context where RFV could be used: RFV of $OHM and RFV of treasury. Your explanation refers to the latter as treasury is mentioned, and I think the first sentence alone is sufficient to explain the concept. See what @Wartull1337 thinks of this

Author: Works for me. Glossary can be kept very simple. If somebody has a straight forward example we can add let me know.

> backing $OHM. For instance $DAI bonds are most efficient for the treasury since
> 100% of the $DAI provided can be accounted for as risk free value.

Reviewer: ```suggestion
backing $OHM.
```
